packages/discord/src/bot.ts

```
@@ -517,6 +619,26 @@ ${formatAPIError(error)}`;
 		return true;
 	}
 
+	private async canUseAuthCommandFromMessage (message: Message): Promise<boolean> {
+		if (!message.inGuild()) {
+			return true;
+		}
+
+		const canAuthenticate
+			= message.member
+			&& message.member.permissions.has(PermissionFlagsBits.Administrator);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)